Source/WebCore/platform/audio/ios/AudioSessionIOS.mm

```diff
@@ -135,7 +135,13 @@ - (void)interruption:(NSNotification *)notification
         return;
     }
 
-    [[PAL::getAVAudioSessionClass() sharedInstance] setHostProcessAttribution:@[ bundleProxy.bundleIdentifier ] error:&error];
+    auto bundleIdentifier = @[ bundleProxy.bundleIdentifier ];
```

Comment:
Shouldn't we check  `bundleProxy.bundleIdentifier` before this line, since inserting a `nil` object in an `NSArray` will crash?

Reply:
Indeed, thanks for pointing that out!